pkg/utils/k8s.go

```diff
@@ -172,7 +190,7 @@ func WaitFor(fn func() error, limit, interval time.Duration) error {
 	for {
 		select {
 		case <-timeout:
-			return fmt.Errorf("timeout with error: %v", err)
+			return fmt.Errorf("timeout with error: %w", err)
```

Comment:
`errors.Wrap()`?

Reply:
errors doesn't have `Wrap()` function, `%w` does it.

Reply:
We started using `"github.com/pkg/errors"` in api-manager and operator-toolkit.  It does the same - it just seems a little more friendly.  It's very subjective of course...

Reply:
Thanks. I don't want to change all the places in this PR.